pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "numalogic"
-version = "0.4a1"
+version = "0.4a2"
```

Let's create a new stable release.
```suggestion
version = "0.4.0"
```

---

Got it!